designcompose/src/main/java/com/android/designcompose/DocServer.kt

```diff
@@ -135,8 +135,12 @@ object DesignSettings {
 
     fun showMessageInToast(msg: String, duration: Int) {
         Log.i(TAG, "Raising toast: $msg")
+        // Toast.makeText dies in AAOS with "display must not be null" on secondary displays.
+        // Just log the message till that issue is resolved.
```

Comment:
Is there a system flag that we could put this behind? Like, is there a flag that marks the system as AAOS instead of regular Android?

Also, is issue present on all versions of AAOS and will previous versions be patched?

Reply:
One option is to add a DesignSettings flag - disableToasts (default false) and is set to true when called from apps that use a secondary screen in AAOS (like the cluster).

I have seen this crash is in S+.